day2.py

```diff
@@ -0,0 +1,109 @@
+import pandas as pd
```

Comment:
For best practice, each module should have a docstring defining the contents of the script
```suggestion
"""Solution for advent of code 2022 day 2.

Script aims to determine who wins rock, paper, scissors.
"""
import pandas as pd
```